linodecli/output/output_handler.py

```diff
@@ -243,8 +244,8 @@ def _get_columns(self, attrs, max_depth=1):
             for col in self.columns.split(","):
                 for attr in attrs:
                     # Display this column if the format string
-                    # matches the column_name or path of this column
-                    if col in (attr.column_name, attr.name):
+                    # matches the path of this column
+                    if col == attr.name:
```

Comment:
This is _technically_ breaking change but it is necessary to resolve the following from giving an unexpected `placement_group.id` output:

```
linode-cli linodes ls --format id
```